tests/test_cp_apr.py

```diff
@@ -238,12 +250,18 @@ def test_cpapr_pqnr(capsys):
     M, _, _ = ttb.cp_apr(sptensorInstance, 2, algorithm="pqnr")
     capsys.readouterr()
     assert np.isclose(M.full().data, ktensorInstance.full().data, rtol=1e-01).all()
-    M, _, _ = ttb.cp_apr(sptensorInstance, 2, algorithm="pqnr", precompinds=False)
+    M, _, output = ttb.cp_apr(sptensorInstance, 2, algorithm="pqnr", precompinds=False)
     capsys.readouterr()
     assert np.isclose(M.full().data, ktensorInstance.full().data, rtol=1e-01).all()
 
+    # Assert that params from previous run can be provided as input
+    M, _, output2 = ttb.cp_apr(sptensorInstance, 2, **output["params"])
```

Comment:
IIRC cp_apr is the slowest of our current tests. Globally, can we also set max iters to a low value throughout the newly added tests since we don't care about the result? In general our test suite runs locally pretty quickly, but trying to prevent gradual bloat.

Reply:
Suggestion: use 10 for the maximum number of iterations

Reply:
Or at least enough to make it through a single pass over all modes/dimensions + 1. This will make sure to test outer iteration and inneriteration.

Reply:
IMO this probably warrants its own issue, so I'll open a new issue for it.